src/components/Header/Header.js

```diff
@@ -120,7 +123,12 @@ const Header = (props) => {
     if (!address) {
       onConnect();
     } else {
```

Comment:
when that else happens, can you explain in a comment

Reply:
it happens when the user is already connected and clicks on the wallet button. added comment in code.